templates/progress.mustache

```diff
@@ -17,16 +17,20 @@
 {{!
     @template report_myfeedback/progress
 
-    This template renders the progress spinner.
+    This template renders a progress GIF.
+
+    Variables required for this template:
+    * progressimg - The URL to the progress image
 
     Example context (json):
     {
-        "progressimg": "/report/myfeedback/pix/progress.gif",
+        "progressimg": "https://localhost/report/myfeedback/pix/progress.gif"
```

Comment:
are you sure this has to be absolute path rather than relative

Reply:
Thank you for this. Obviously this is a flaw...